rest-api/tools/import_organizations.py

```diff
@@ -121,16 +137,93 @@ def _entity_from_row(self, row):
-
-    self.organization_dao = OrganizationDao()
     args = parser.parse_args()
+    self.organization_dao = OrganizationDao()
     self.geocode_flag = args.geocode_flag
     self.ACTIVE = SiteStatus.ACTIVE
     self.status_exception_list = ['hpo-site-walgreensphoenix']
+    self.instance = args.instance
+    self.creds_file = args.creds_file
+    self.new_sites_list = []
+    self.project = None
+    if args.project:
+      self.project = args.project
+
+    if self.project in ENV_LIST:
+      self.environment = ' ' + self.project.split('-')[-1].upper()
+    else:
+      self.environment = ' ' + ENV_TEST.split('-')[-1].upper()
+
+    super(SiteImporter, self).__init__('site', SiteDao(), 'siteId', 'googleGroup',
+                                       [SITE_ORGANIZATION_ID_COLUMN, SITE_SITE_ID_COLUMN,
+                                        SITE_SITE_COLUMN, SITE_STATUS_COLUMN + self.environment,
+                                        ENROLLING_STATUS_COLUMN + self.environment,
+                                        DIGITAL_SCHEDULING_STATUS_COLUMN + self.environment])
+
+  def run(self, filename, dry_run):
+    super(SiteImporter, self).run(filename, dry_run)
+    insert_participants = False
+    if not dry_run:
+      if self.environment:
+        if self.environment.strip() == 'STABLE' and len(self.new_sites_list) > 0:
+          from googleapiclient.discovery import build
+          service = build('appengine', 'v1', cache_discovery=False)
+          request = service.apps().services().versions().list(appsId='pmi-drc-api-test',
+                                                                    servicesId='default')
+          versions = request.execute()
+
+          for version in versions['versions']:
+            if version['servingStatus'] == 'SERVING':
+              _id = version['id']
+              request = service.apps().services().versions().instances().list(
+                                                                      appsId='all-of-us-rdr-stable',
+                                                                      servicesId='default',
+                                                                      versionsId=_id)
+              instances = request.execute()
+
+              for instance in instances['instances']:
+                sha = instance['name'].split('/')[-1]
+                delete_instance = service.apps().services().versions().instances().delete(
+                                                                      appsId='all-of-us-rdr-stable',
+                                                                      servicesId='default',
+                                                                      versionsId=_id,
+                                                                      instancesId=sha)
+
+                response = delete_instance.execute()
+
+                if response['done']:
+                  insert_participants = True
```

Comment:
Rather than having the insert_participants flag, you could probably just move insert_new_participants call into this if?